### ISSUE TITLE
Sparse documentation on exposed method 

### ISSUE THREAD

**Issue by psychoslave** [grosser/fast_gettext#104]
The documentation give some hints that there is actually more than `_` that is usable as exposed method. Namely it talks about `N_`, `n_` and `s_`. But it doesn't list them up to describe thoroughly each one. Even in the comments of `lib/fast_gettext/translation.rb`, explanations are rather sparse.

The documentation should have a dedicated section explaining what each of the following method is for:
```ruby
def self.included(klas) ;end
def _(key, &block) ;end
def n_(*keys, &block) ;end
def s_(key, separator=nil, &block) ;end
def N_(translate) ;end
def Nn_(*keys) ;end
def ns_(*args, &block) ;end
def self.included(klas) ;end
def _in_domain domain ;end
def d_(domain, key, &block) ;end
def dn_(domain, *keys, &block) ;end
def ds_(domain, key, separator=nil, &block) ;end
def dns_(domain, *keys, &block) ;end
def D_(key) ;end
def Dn_(*keys) ;end
def Ds_(key, separator=nil) ;end
def Dns_(*keys) ;end
```

The documentation should also mention that `msgctx` are not supported so far, although there is an open issue on that: https://github.com/grosser/fast_gettext/issues/87

**Comment by psychoslave** [comment]
@grosser would you mind writing the documentation?

@connorshea any news about making a PR for `msgctx` support?

**Comment by grosser** [comment]
feels free to open PRs for where docs are missing,
since this is meant for people that know what gettext is I'm keeping it
light

**Comment by rgraff** [comment]
@grosser I think it'd be nice to match all the exposed methods of ruby-gettext (e.g. `gettext` , `sgettext`).  Ok for me to add them and then update the README?

**Comment by rgraff** [comment]
Most of this is addressed in #106

**Comment by rgraff** [comment]
@grosser I think you can close this issue. Everything was addressed in the last few PRs
